File: hello/py/hello.py
import time as t
import skilstak.colors as c
def print_plain(message):
    """Prints Hello World to the Console"""
    print(c.cl + message + c.x)

def print_multi(message):
    """Lights up text in random colors like a Christmas Tree"""
    while True:
        print(c.clear + c.multi(message))
        t.sleep(0.5)
def print_color(message):
    """Prints hello World in a random color"""
    print(c.cl + c.rc() + message + c.x)

def print_forever(message):
    """Prints Hello World in random colors all over the screen"""
    print(c.cl)
    while True:
        print(c.rc() + message + c.reset, end = " ")

def parse_args():
    import sys
    p = {
            "who":"world",
            "option": ""
        }
    
    if len(sys.argv) > 2:
       p['option'] = sys.argv[1]
       p["who"] = sys.argv[2]
    elif len(sys.argv) == 2:
        if sys.argv[1].startswith("-"):
           p['option'] = sys.argv[1]
        else:
            p['who'] = sys.argv[1]
        return p

File: go/src/hello/hello.go
package hello

import (
	"fmt"
	c "github.com/skilstak/go/colors"
	s "strings"
	//i "github.com/skilstak/go/input"
	"os"
	t "time"
)

func PrintPlain(message string) {
	//Prints "Hello World!" to the command line.
	fmt.Println(c.Clear + "Hello " + message + c.X)
}
func PrintRandom(message string) {
	//Prints "Hello World" to the command line in random color.
	fmt.Println(c.Clear + c.Rc() + "Hello " + message + c.X)
}
func PrintMulti(message string) {
	//Prints Hello World to the command line with each character a different color.
	for {
		fmt.Println(c.Multi("Hello " + message + c.Clear))
		t.Sleep(500 * t.Milliseconds)
	}
}
func PrintForevah(message string) {
	//Prints "Hello World" in a random color forever in a loop.
	for {
		fmt.Print(c.Rc() + "Hello " + message + c.X)
	}
}

func ParseArgs() {
	//Parses arguments from the command line
	name := "YOU"
	option := ""
	if len(os.Args) > 2 {
		if s.HasPrefix(os.Args[1], "-") {
			option = os.Args[1]
			name = os.Args[2]
		}
	}
	if len(os.Args) == 2 {
		name = os.Args[1]
	}

}
